contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -40,7 +41,7 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
     // Tracks the outstanding receipts to send back to a given subnet in subsequent messages sent to it.
     // Key is the subnet ID of the other subnet, and the value is a queue of pending receipts for messages
     // we have received from that subnet.
-    mapping(bytes32 => ReceiptQueue) public outstandingReceipts;
+    mapping(bytes32 => ReceiptQueue.TeleporterMessageReceiptQueue) public outstandingReceipts;
```

Comment:
when setting this state variable, and the fields within set to their default value? How about for the struct `TeleporterMessageReceipt` inside the receipt queue's data?

Reply:
yes, it will be the default value if it does not exist in the map. The fields in the struct are initialized as default value.
```
struct TeleporterMessageReceiptQueue {
        uint256 first;
        uint256 last;
        mapping(uint256 index => TeleporterMessageReceipt) data;
}
```
`first`, `last`, `data` are initialized as the default value. When we access `data[idx]`, if it does not exist, it should be default value of struct `TeleporterMessageReceipt`.